test/rs/src/bin/test_server.rs

```diff
@@ -16,14 +16,12 @@
 // under the License.
 
 use clap::{clap_app, value_t};
-use env_logger;
```

Comment:
I'm a little surprised that we don't need the use declaration anymore.

Reply:
@allengeorge this is clippy::single_component_path_imports

```
error: this import is redundant
```